org.eclipse.jdt.core/.settings/.api_filters

```diff
@@ -238,4 +238,20 @@
             </message_arguments>
         </filter>
     </resource>
+    <resource path="search/org/eclipse/jdt/core/search/IJavaSearchDelegate.java" type="org.eclipse.jdt.core.search.IJavaSearchDelegate">
```

Comment:
Is the API filter really necessary ? I looked in the build failures. It was failing with : 
```
Invalid @since 3.42 tag on org.eclipse.jdt.core.search.IJavaSearchDelegate; expecting @since 3.41
API problem: search/org/eclipse/jdt/core/search/IJavaSearchDelegate.java
```

but I think the reason for that is because jdt.core has already been bumped to 3.41 in Eclipse 4.35. It was released as 3.40 in Eclipse 4.34. See https://www.eclipse.org/downloads/download.php?file=/eclipse/updates/4.34/R-4.34-202411201800/plugins/org.eclipse.jdt.core_3.40.0.v20241118-1641.jar . So `@since 3.41` is the "new" version you want to use.

Reply:
The API filters are because of non-API parameters.